imports/startup/client/clientError.js

```diff
@@ -0,0 +1,50 @@
+const reportToServer = function(error) {
+	var report =
+		{ name: error.name
+		, message: error.message
+		, location: window.location.href
+		, tsClient: new Date()
+		};
+	Meteor.call('clientError', report);
+}
+
+window.addEventListener("error", function(event) {
+	reportToServer(event.error);
+});
+
+var buffer = [];
+var discriminatoryReporting = function(arguments) {
+	var msg = arguments[0];
+
+	// "Exception from Tracker recompute function:"
+	if (msg.indexOf("Exception ") === 0) {
+		// Boring
+		return;
+	}
+
+	// "Error: No such function: ..."
+	if (msg.indexOf("Error:") === 0) {
+		buffer.push(msg);
+		return;
+	}
+
+	// "Blaze.View.prototy..."
+	if (msg.indexOf("Blaze.") === 0) {
+		// There's a template name in there right?
+		var templateNames = /Template\.[^_]\w+/g;
+		buffer.push(msg.match(templateNames).join(','));
+		reportToServer(
+			{ name: 'TemplateError'
+			, message: buffer.join('; ')
+			}
+		);
+		return;
```

Comment:
Log all the things! If we arrive here it means it's somethign we don't know about, so we should.